pt-BR/action_view_overview.md

```diff
@@ -370,34 +370,35 @@ Supposing we use the same `_box` partial from above, this would produce the same
 View Paths
 ----------
 
-When rendering a response, the controller needs to resolve where the different
-views are located. By default it only looks inside the `app/views` directory.
+Ao renderizar uma resposta, o controler precisa resolver onde as diferentes
+views estão localizadas. Por padrão, ele só olha dentro do diretório `app/views`.
```

Comment:
```suggestion
Ao renderizar uma resposta, o *controller* precisa resolver onde as diferentes
views estão localizadas. Por padrão, ele só olha dentro do diretório `app/views`.
```

Reply:
Normalmente marcamos algumas coisas como itálico como descrito no [Manual de Tradução](https://github.com/campuscode/rails-guides-pt-BR/blob/main/TRANSLATION_MANUAL.md)